modules/DMZ/models/model_properties.py

```diff
@@ -0,0 +1,10 @@
+# This file will be part of forcing models to have specific properties that govern
+# their usage.
+
+
+class ModelProperties(object):
+
+    def __init__(self, regression=False, online=False):
+        self._classification = not regression  # I don't know if I should get fancy...
```

Comment:
@asclines Like in an XML file? No we haven't, for now I'm just initializing per model